releasenotes/notes/fixed_1q_decomp-9a295d248c04b862.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    For classes (such as :class:`.LayerFidelity`) that use the ``clifford_utils``
```

Comment:
This is good to have in the release notes, but could we document `1Q_fixed` somewhere in the documentation? The hard part is that I don't know where to say to document it. `synthesis_method` is documented as defaulting to `rb_default` in a few places but there is no discussion of why you would want to change it or what other option you might choose. Maybe just a couple sentences in the `randomized_benchamrking/__init__.py` docstring that shows up at https://qiskit-community.github.io/qiskit-experiments/apidocs/mod_randomized_benchmarking.html?

Reply:
in `__init__.py` do I just add something like

---

Notes
=========

By default for n<=2 Cliffords these rb methods will transpile using `optimization_level=1`. There are a few special options:
-  `synthesis_method='basis_only'` will use `optimization_level=0`
-  `synthesis_method='1Q_fixed` will use a rz-sx-rz-sx-rz decomposition for the 1Q Cliffords and the default for the 2Q cliffords. This is most relevant for `LayerFidelity` experiments because it will keep a fixed structure.